src/frontend/qt_sdl/Platform.cpp

```diff
@@ -127,6 +131,24 @@ void StopEmu()
     emuStop();
 }
 
+void LogMessage(const char* format, ...)
+{
+    va_list args;
+    va_start(args, format);
+    if (Config::LogToFile)
+    {
+        if (!LogFile)
+        {
+            LogFile = fopen(Config::LogFileLocation, "a+");
```

Comment:
a+ seems to be dangerous, because melonDS can spit out a lot of data

Reply:
should i change to w+?

Reply:
only w we don't need to read

Reply:
done